app/views/posts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with( url: user_posts_path(@user), method: :post) do |form| %>
+  <%= form.label :title %>
+  <%= form.text_field :title %>
+
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit 'Create Post' %>
+<% end %>
```

Comment:
##Required

- [ ]  Kindly make this form to be displayed as in the wireframes. It is required to accept input for creating a post